.github/pull_request_template.md

```diff
@@ -6,3 +6,4 @@
 
 - [ ] I have updated `CHANGELOG.md` if required.
 - [ ] I have updated documentation if required.
+- [ ] I have regenerated API dump via `./gradlew apiDump` task.
```

Comment:
Perhaps as part of CI we could also call `apiDump` and compare it against the PR? We could then fail the build if there is a discrepancy.

That way we don't need this checkbox, and we will avoid the situation where someone regenerates it on their PR and several other PR changes are added into that PR, making reviewing confusing!

Reply:
`apiCheck` task is already part of `build` task chain.